utils/bytes_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package utils
+
+import (
+	"math"
+	"math/rand"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestIntSize(t *testing.T) {
+	require := require.New(t)
+
+	require.Contains([]int{32, 64}, intSize)
+	if intSize == 32 {
+		require.Equal(math.MaxInt32, math.MaxInt)
+	} else {
+		require.Equal(math.MaxInt64, math.MaxInt)
+	}
+}
+
+func TestBytesPool(t *testing.T) {
+	require := require.New(t)
+
+	p := NewBytesPool()
+	for i := 0; i < 128; i++ {
+		bytes := p.Get(i)
+		require.NotNil(bytes)
+		require.Len(*bytes, i)
+		p.Put(bytes)
+	}
+}
+
+func TestBytesPoolMaxInt(t *testing.T) {
+	require := require.New(t)
+
+	p := NewBytesPool()
+	bytes := p.Get(math.MaxInt32)
```

Comment:
q: does this allocate all of this memory?
Not sure we ever need this much, nor that it's the right thing to do in a UT.
Should we consider a smaller max size? Or make it configurable?

Reply:
The `BytesPool` supports allocating slices in the range `[0, math.MaxInt]` (which is the same as `make`.

I added this test when I was hardcoding `intSize` to be `32`. I'm going to remove this test as it really doesn't make sense without that modification.